src/woo_publications/api/urls.py

```diff
@@ -4,16 +4,22 @@
 from drf_spectacular.views import SpectacularJSONAPIView, SpectacularRedocView
 from rest_framework import routers
 
+from woo_publications.metadata.api.views.category import InformationCategoryViewset
+
 app_name = "api"
 
 router = routers.DefaultRouter()
+router.include_root_view = False
+
+router.register("informatie-categorie", InformationCategoryViewset)
```

Comment:
```suggestion
router.register("informatiecategorieen", InformationCategoryViewset)
```
it's a single word in Dutch, and for base/collection endpoints we use plural form

Reply:
not resolved